shanoir-ng-datasets/src/main/java/org/shanoir/ng/download/WADODownloaderService.java

```diff
@@ -107,7 +117,7 @@
 	 */
 	public void downloadDicomFilesForURLs(final List<URL> urls, final File workFolder, String subjectName) throws IOException, MessagingException {
 		for (Iterator iterator = urls.iterator(); iterator.hasNext();) {
-			String url = ((URL) iterator.next()).toString();
+			String url = ((URL) iterator.next()).toString() + "&";
```

Comment:
What is it for ? :)

Reply:
I don't know I removed it and it seems to work